github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits-1.38.yaml

```diff
@@ -66,3 +62,116 @@ presubmits:
           resources:
             requests:
               memory: "29Gi"
+presubmits:
+  kubevirt/containerized-data-importer:
+  - name: pull-cdi-unit-test-1.38
+    branches:
+      - release-v1.38
+    cluster: prow-workloads
```

Comment:
Please add the `context` field here. Example: https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.58.yaml#L9

Reply:
```suggestion
    cluster: prow-workloads
    context: pull-cdi-unit-test
```